clang_delta/EmptyStructToInt.cpp

```diff
@@ -213,6 +259,54 @@ void EmptyStructToInt::HandleTranslationUnit(ASTContext &Ctx)
     TransError = TransInternalError;
 }
 
+void EmptyStructToInt::handleOneRecordDecl(const RecordDecl *RD,
+                                           const RecordDecl *BaseRD,
+                                           const FieldDecl *FD,
+                                           unsigned int Idx)
+{
+  // Skip field if it is not of the type of one of
+  // the structs that are affected by the change
+  // Works recursively through chained structs
+  if (!RecordDeclToField[BaseRD] && BaseRD != TheRecordDecl) {
```

Comment:
Seems we could remove ```!RecordDeclToField[BaseRD] &&&```? Do you have an example that would fail if we skip this check?